common/lib/actions/poll_details.dart

```diff
@@ -7,10 +7,10 @@ part 'poll_details.g.dart';
 
 @JsonSerializable()
 class PollDetailsResponse {
-  final Poll poll;
-  final List<Comment> comments;
+  final Poll parent;
```

Comment:
Do these fields need to be renamed? `PollDetailsResponse` should just contain the top-level comments and there should be another request/response to fetch replies.

Reply:
I did this so I could refer to polls and comments as parents; comments and replies as messages.